.github/workflows/ci.yml

```diff
@@ -127,10 +127,18 @@ jobs:
           cache: 'yarn'
       - name: Install dependencies
         run: yarn --immutable
+      - name: Pre-Cypress (build all dependencies)
+        run: yarn test:ci-pre:vue
       - name: Cypress run
-        uses: cypress-io/github-action@v5.5.0
+        uses: cypress-io/github-action@v5.6.1
         with:
-          command: yarn test:ci:vue
+          browser: chrome
+          working-directory: apps/test/vue
+          component: true
```

Comment:
Question:
Why no use the custom command (`yarn test:ci:vue`) like before? It already did the coverage generation and could handle dependencies build, no?

Reply:
because tests run 12 minutes instead of 6 minutes in total (whole step). They do some optimisation in their side in docker I'd say